ci/asana-create-tasks.js

```diff
@@ -11,7 +11,7 @@ const releaseUrl = process.env.RELEASE_URL || 'https://example.com/';
 const releaseNotesRaw = process.env.RELEASE_NOTES || '<EMPTY RELEASE NOTES>';
 const releaseNotes = md.render(releaseNotesRaw);
 
-const templateTaskGid = '1206774921409831';
+const templateTaskGid = '1209334124753471'; /* TODO: Replace with original template ID after monorepo migration */
```

Comment:
This points to an updated copy of the template that uses a single `Apple` platform subtask. The template will be updated when the monorepo migration is happening (within the next two weeks).